File: hw/maisincho/u3/HW19AbstractFactory/AbstractFactory/src/main/java/ec/edu/espe/abstractFactory/view/ClientApp.java
package ec.edu.espe.abstractFactory.view;

import ec.edu.espe.abstractFactory.controller.GUIFactory;
import ec.edu.espe.abstractFactory.model.Button;

/**
 *
 * @author Isaac Maisincho Crafters_Market DCCO ESPE
 */
public class ClientApp {

    public static void main(String[] args) {
        GUIFactory factory = GUIFactory.getFactory();
        Button button = factory.createButton();
        button.caption = "Play";
        button.paint();
        button.caption = "Exit";
        button.paint();
    }
}

File: hw/maisincho/u3/HW19AbstractFactory/AbstractFactoryPy/ClientApp.py
from controller.GUIFactory import GUIFactory

def main():
    factory = GUIFactory.get_factory()

    button = factory.create_button()
    button.caption = "Play"
    button.paint()

    button.caption = "Exit"
    button.paint()

if __name__ == "__main__":
    main()